packages/core/src/components/TextField/TextField.tsx

```diff
@@ -241,12 +243,16 @@ const TextField: VibeComponent<TextFieldProps, unknown> & {
     }, [disabled, clearOnIconClick, onIconClick, currentStateIconName, controlled, onChangeCallback, clearValue]);
 
     const validationClass = useMemo(() => {
+      if (maxLength && inputValue.length > maxLength) {
```

Comment:
Weird weird edge case here, but if someone specified `maxLength={0}`, what do we expect our functionality to be? In an if statement it'll be interepreted as false, so it might look weird here. Just want to make sure it's considered!

Reply:
Yay falsy values 🥳 

Edge-casiest of edge cases indeed, I'll do an undefined check for correctness sake. Good spot 🙌